docs/source/tutorials/nemo1d.rst

```diff
@@ -0,0 +1,123 @@
+
+.. _nemo:
+
+#############################
+NEMO-ERSEM in a water column 
+#############################
+
+This tutorial demonstrates how to configure and run NEMO 1D (water-column) reference configuration C1D_PAPA with ERSEM biogeochemistry. This example is a demonstration of the concept, as it combines hydrodynamics of ocean station `PAPA <https://www.pmel.noaa.gov/ocs/Papa>`_ in the North Pacific Ocean with biogeochemistry of `L4 station <https://www.westernchannelobservatory.org.uk/>`_ in the Western English Channel. Users are encouraged to modify this example to fit their own purposes.
+
+We recommend to read the brief description of the NEMO `C1D_PAPA configuration <https://forge.ipsl.jussieu.fr/nemo/chrome/site/doc/NEMO/guide/html/cfgs.html#c1d-papa>`__ before continuing with this tutorial.
+
+Step 1: Obtaining the code
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The input data contained in INPUTS_C1D_PAPA_v4.0.tar file must be obtained from `NEMO Reference configurations inputs <https://zenodo.org/record/1472245#.Yt6_QIzMKEI>`__ repository on Zenodo  and unpacked into the working directory.
+
+NEMO4 code base with FABM support can be obtained in the corresponding `repository <https://github.com/pmlmodelling/NEMO4.0-FABM>`__. The next step is to download `FABM <https://github.com/fabm-model/fabm>`__ and `ERSEM <https://github.com/pmlmodelling/ersem>`__. Finally, I/O server `XIOS-2.5 <https://forge.ipsl.jussieu.fr/nemo/chrome/site/doc/NEMO/guide/html/install.html#extract-and-install-xios>`__ must be downloaded and `installed <https://forge.ipsl.jussieu.fr/ioserver/>`__. The compiled xios_server.exe executable should be copied into the working directory.
+
+
+Step 2: Compiling the code
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+First, FABM must be compiled with ERSEM support, specifying nemo as a physical host. The following commands can be run in a command line or wrapped into an executable file for easy recompilation at any future point:
```

Comment:
Maybe instead of "physical host" say that it is the "physical ocean model" or similar instead? Up to you though